### ISSUE TITLE
Cannot install this module

### ISSUE THREAD

**Issue by Kerumen** [Apercu/adyen-encrypt#3]
Hi,

I tried to `npm i -S adyen-encrypt` this module but this failed with an error:

```
> adyen-encrypt@0.1.8 postinstall /Users/yann/git/us-vanderbilt/node_modules/adyen-encrypt
> npm run sjcl


> adyen-encrypt@0.1.8 sjcl /Users/yann/git/us-vanderbilt/node_modules/adyen-encrypt
> (cd ./node_modules/sjcl && ./configure --with-codecBytes && make)

sh: line 0: cd: ./node_modules/sjcl: No such file or directory
```

PS: I have Java on my machine. But I guess this error is caused by a path not found.

**Comment by balthazar** [comment]
Certainly related https://github.com/bitwiseshiftleft/sjcl/issues/319

**Comment by Kerumen** [comment]
This is not related, I'm not getting this error.
There is a path issue with your module. It can't find the directory `./node_modules/sjcl` because the script is ran from the root of your module and since npm@3 and all deps flattened, you'll have to do `cd ../sjcl` to target the right directory.

I can PR if necessary.

By the way, the issue you linked will still remains until fixed in `sjcl`.

**Comment by balthazar** [comment]
I am aware.. and judging by the answer you got I don't think it will.

There is two issues then, because even if it would be the right path for you, sjcl won't be able to compile. It's a bit more complicated than changing the path, I'll try to land something today.

**Comment by Kerumen** [comment]
Cool thanks!

**Comment by balthazar** [comment]
Please try using `yarn add adyen-encrypt@next`. I don't have any easy access to a repo using this, you're my only hope Obi-Yann Keruby.

![hope](https://cdn.pbrd.co/images/flZouhHr3.png)

**Comment by Kerumen** [comment]
It works like a charm. Thanks for the quick fix!

**Comment by balthazar** [comment]
Noice, published as `0.2.0`
